.github/workflows/ci-detox-android.yml

```diff
@@ -16,12 +16,6 @@
 #       - name: Checkout
 #         uses: actions/checkout@83b7061638ee4956cf7545a6f7efe594e5ad0247 # pin@v3.5.1
 
-#       # Needed to resolve any private git based dependencies like https://github.com/mattrglobal/blstrs
-#       - name: Setup SSH Agent
```

Comment:
is it commented out on purpose?

Reply:
Yeah those already existed take [ci-detox-android](https://github.com/mattrglobal/pairing_crypto/blob/2e0901732f00b05f58b16ae1aa067de697e97eb6/.github/workflows/ci-detox-android.yml) for example they have commented out the entire workflow in this change
https://github.com/mattrglobal/pairing_crypto/commit/b35fd0ede6e2b4d4b7e88976fabc38bb90154c77